src/store/drawing/reducer.ts

```diff
@@ -7,14 +8,21 @@ import {
     redoDrawingPathGroupAction,
     resetDrawingAction,
     setDrawingLocationAction,
+    setDrawingStrokeAction,
     undoDrawingPathGroupAction,
 } from './actions';
 import { DrawingState } from './types';
 
+export const defaultStroke = {
+    color: bdlBoxBlue,
+    size: 4,
```

Comment:
Is this size used anywhere else? If it is, maybe we can use the same reference rather than a hardcoded constant here?

Reply:
No, this is the only place.